src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -92,6 +93,22 @@ public static Description parseDescription(String description) throws ParseExcep
         return new Description(trimmedDescription);
     }
 
+    /**
+     * Parses a {@code String priority} into a {@code Priority}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code description} is invalid.
+     */
+    public static Priority parsePriority(String priority) throws ParseException {
+        requireNonNull(priority);
+        String trimmedPriority = priority.trim();
```

Comment:
Is it better if we make priority case insensitive here? This could be achieved by convert the string to lower case here.

Reply:
Good idea, will do so!